atlasdb-backup/src/test/java/com/palantir/atlasdb/backup/AtlasRestoreServiceTest.java

```diff
@@ -58,9 +60,12 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class AtlasRestoreServiceTest {
```

Comment:
How about if we ask to repair for service ID A and namespace U in a request, then immediately request a repair for service ID B and namespace U in the next? Do we support that for the same restore service?

Reply:
The request will fail iff the backup IDs are different.